lib/osf-components/addon/components/registries/schema-chunk/template.hbs

```diff
@@ -0,0 +1,40 @@
+{{yield (hash
+    page-heading=(
+        component
+        'registries/schema-chunk/page-heading'
+    )
+    section-heading=(
+        component
+        'registries/schema-chunk/section-heading'
+    )
+    subsection-heading=(
+        component
+        'registries/schema-chunk/subsection-heading'
+    )
+    paragraph=(
+        component
+        'registries/schema-chunk/paragraph'
+    )
+    label=(
+        component
+        'registries/schema-chunk/label'
+        answerId=@answerId
+    )
+    text=(
+        component
+        'registries/schema-chunk/text'
+        answerId=@answerId
+        changeset=@changeset
+        disabled=this.disabled
+        shouldShowMessages=this.shouldShowMessages
+    )
+    textarea=(
+        component
+        'registries/schema-chunk/textarea'
+        chunkId=@chunkId
```

Comment:
Is `chunkId` needed? I don't see it used in the `schema-chunk/textarea` component.

Reply:
BTW, what is `chunkId`? How is it different from `answerId`?

Reply:
So, chunkId is used to categorize parts of a schema into chunks and answerId is placed on anything that requires an answer for easier access.  That being said, chunkId is definitely not going to be used after the schema is chunked out, so it's not needed here.  I'll remove that.